<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_login</class>
 <widget class="QDialog" name="Dialog_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>101</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icon/Bokehlicia-Captiva-Rocket.ico</normaloff>../Icon/Bokehlicia-Captiva-Rocket.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_Username">
           <property name="text">
            <string>Username</string>
           </property>
           <property name="buddy">
            <cstring>le_username</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Pass">
           <property name="text">
            <string>Password</string>
           </property>
           <property name="buddy">
            <cstring>le_password</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="le_username">
           <property name="text">
            <string>myusername</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_password">
           <property name="text">
            <string>mypassword</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_Cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_Login">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_username</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>pb_Login</tabstop>
  <tabstop>pb_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>le_password</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>le_username</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Login</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_login</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
